api/NetworkInformation.json

```diff
@@ -249,7 +249,7 @@
               "version_added": false
             },
             "chrome_android": {
-              "version_added": false
+              "version_added": "38"
```

Comment:
This changes the Chrome for Android version, was that accidental given the PR description?

Reply:
https://github.com/mdn/browser-compat-data/commit/c5e72068ff0c2dc7b6aae633e425751c0024ac46

Initially I removed this item and then found it's also in Blink.

Reply:
Yep, looks like it is enabled: https://source.chromium.org/chromium/chromium/src/+/master:third_party/blink/renderer/platform/runtime_enabled_features.json5;l=1345;drc=2e80c2498eb1d8c67b36ee6b27fa6381eb8aacab